packages/charts/src/charts/bar/utils/mark.utils.test.ts

```diff
@@ -67,16 +68,43 @@ describe('@keen.io/charts - <BarChart /> utils', () => {
       });
     });
 
-    test('should calculate position for "horizontal" layout', () => {
+    test('should calculate position for "horizontal" layout, positive value', () => {
       const result = calculateMarkPosition({
         ...barProperties,
         layout: 'horizontal',
+        positiveValue: true,
       });
 
       expect(result).toEqual({
         x: 110,
         y: 60,
       });
     });
+
+    test('should calculate position for "vertical" layout, negative value', () => {
```

Comment:
Please, re-phrase test cases sentences. Especially in test case You used `negative value` and variable for `calculateMarkPosition` is named `positiveValue`.

`calculates correctly position in "vertical" layout when value is negative`.